src/test/java/bio/terra/service/dataset/DatasetAzureIntegrationTest.java

```diff
@@ -300,6 +306,42 @@ public void datasetIngestFileHappyPath() throws Exception {
         dataRepoFixtures.getFileByName(steward(), datasetId, file2.getTargetPath());
     assertThat("Test retrieve file by path", file2Model.getFileId(), equalTo(file2.getFileId()));
 
+    // ingest via control file
+    String flightId = UUID.randomUUID().toString();
+    String controlFileBlob = flightId + "/file-ingest-request.json";
+    List<BulkLoadFileModel> bulkLoadFileModelList = new ArrayList<>();
+    bulkLoadFileModelList.add(
+        new BulkLoadFileModel()
+            .mimeType("text/plain")
+            .sourcePath(
+                String.format("%s/%s", blobIOTestUtility.getSourceContainerEndpoint(), sourceFile))
```

Comment:
It looks like we do this a lot. Would it be worth adding a method to `BlobIOTestUtility`?
```suggestion
            .sourcePath(blobIOTestUtility.createPath(sourceFile))
```

```
public class BlobIOTestUtility {
  [...]
  public String createPath(String blobName) {
    return String.format("%s/%s", getSourceContainerEndpoint(), blobName);
  }
```

If you added an overloaded form of `createPath()` that accepts a query argument, it looks like you could make `getSourceContainerEndpoint()` `private`, which would be nice.

Reply:
Nice, yeah, this is used a lot throughout the test. I think I've changed them all to use the new helper method.